modules/javafx.graphics/src/main/java/com/sun/glass/utils/NativeLibLoader.java

```diff
@@ -111,9 +111,16 @@ public static synchronized void loadLibrary(String libname, List<String> depende
     }
 
     private static void loadLibraryInternal(String libraryName, List<String> dependencies, Class caller) {
-        // Look for the library in the same directory as the jar file
-        // containing this class.
-        // If that fails, then try System.loadLibrary.
+        // The search order for native library loading is:
+        // - try to load the native library from the same folder as this jar
+        //   (only on non-modular builds)
+        // - if the native library comes bundled as a resource it is extracted
+        //   and loaded
+        // - the jna.library.path is searched for the library in definition
```

Comment:
That should be `java.library.path`

Reply:
Sorry, I had much more need to write `jna.library.path`, than `java.library.path`. Updated accordingly.